l2geth/rollup/sync_service.go

```diff
@@ -1151,6 +1154,75 @@ func (s *SyncService) verifyFee(tx *types.Transaction) error {
 	return nil
 }
 
+func (s *SyncService) VerifiedTxCount() (uint64, error) {
+	var pendingTxCount uint64
+	pendingTxs, err := s.txpool.Pending()
+	if err != nil {
+		return 0, err
+	}
+	for _, txs := range pendingTxs {
+		for _, tx := range txs {
+			if err := s.VerifyFee(tx); err == nil {
+				pendingTxCount++
+			}
+		}
+	}
+	return pendingTxCount, nil
+}
+
+func (s *SyncService) VerifyFee(tx *types.Transaction) error {
+	if tx == nil {
+		return errors.New("nil transaction passed to ValidateAndApplySequencerTransaction")
+	}
+	s.txLock.Lock()
+	defer s.txLock.Unlock()
+	if err := s.verifyFee(tx); err != nil {
+		return err
+	}
+	return nil
+}
+
+// ApplyGasPriceTxs apply tx for update gasPrice
+func (s *SyncService) ApplyUpdateGasPriceTxs() {
+	for {
+		if len(s.updateGasPriceTxPool) == 0 {
+			break
+		}
+		tx := <-s.updateGasPriceTxPool
```

Comment:
```
for {
		if len(s.updateGasPriceTxPool) == 0 {
			break
		}
        tx := <-s.updateGasPriceTxPool
```
How about replacing the above code below?
```
for tx := range s.updateGasPriceTxPool {
```

Reply:
when len(s.updateGasPriceTxPool) == 0 must break,Otherwise, the process will be blocked